Outdated build instructions, build broken with dotnet-sdk 3.1

Build error with dotnet-sdk 3.1:
```
SDK Version: 3.1.415
...
/nix/store/bkil2pn0r10zy1i3919qg36nn2gb81qw-dotnet-sdk-3.1.415/sdk/3.1.415/Microsoft.Common.CurrentVersion.targets(1177,5): error MSB3971: The reference assemblies for ".NETFramework,Version=v6.0" were not found. You might be using an older .NET SDK to target .NET 5.0 or higher. Update Visual Studio and/or your .NET SDK. [/btcpayserver/BTCPayServer/BTCPayServer.csproj]

Build FAILED.
```

Affected pages:
```
./docs/Deployment/ManualDeploymentExtended.md  (Many references)
./docs/Deployment/ManualDeployment.md          (Many references)
./docs/Contribute/DevCode.md:                  (1. [.NET Core SDK 3.1+](https://dotnet.microsoft.com/download))
./docs/Development/Altcoins.md                 (Instead, install [.NET Core 3.1 SDK](https://www.microsoft.com/net/download/windows))
./docs/Development/LocalDev.md                 ([.NET Core 3.1 SDK](https://dotnet.microsoft.com/download))
```

--- a/docs/Contribute/DevCode.md
+++ b/docs/Contribute/DevCode.md
@@ -17,7 +17,7 @@ If you are looking for how to make a simple code change such as fixing a typo or
  Software to install to follow this guide:
 
  1. [Visual Studio Community Edition](https://visualstudio.microsoft.com/downloads/)
- 1. [.NET Core SDK 3.1+](https://dotnet.microsoft.com/download)
+ 1. [.NET Core SDK 6.0+](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
  1. [Docker Desktop](https://www.docker.com/products/docker-desktop)
  1. PowerShell (included in Windows OS)
  1. [GitBash](https://gitforwindows.org/)

--- a/docs/Deployment/ManualDeployment.md
+++ b/docs/Deployment/ManualDeployment.md
@@ -48,15 +48,24 @@ sudo tar -xzvf bitcoin.tar.gz -C /usr/local/bin --strip-components=2 "bitcoin-$B
 rm bitcoin.tar.gz
 ```
 
-### 2) Install .NET Core SDK 3.1
-On my ubuntu 18.04 (See [these instructions](https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1804) or [here](https://dotnet.microsoft.com/download) for different OS).
+### 2) Install .NET 6.0 SDK
+
+On my Ubuntu 20.04 (See [these instructions](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2004-) or [here](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) for different OS).
 
 ```bash
-wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
+# Add Microsoft package repository
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
 sudo dpkg -i packages-microsoft-prod.deb
-sudo apt-get install apt-transport-https
+rm packages-microsoft-prod.deb
+
+# Install the SDK
 sudo apt-get update
-sudo apt-get install -y dotnet-sdk-3.1
+sudo apt-get install -y apt-transport-https
+sudo apt-get update
+sudo apt-get install -y dotnet-sdk-6.0
+
+## Check
+dotnet --version
 ```
 
 ### 3) Install NBXplorer

--- a/docs/Deployment/ManualDeploymentExtended.md
+++ b/docs/Deployment/ManualDeploymentExtended.md
@@ -707,19 +707,24 @@ To connect another `bitcoind` instance to your new node:
 
 ##### 🚚 Install
 
-##### 1. Install .Net Core
+##### 1. Install .NET 6.0 SDK
 
-Check [follow the install instuctions](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2004-) (.Net Core 3.1.6 at the time of writing)
+[Follow the install instuctions](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#2004-):
 
 ```bash
-~$ wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-~$ sudo dpkg -i packages-microsoft-prod.deb
-~$ sudo apt update
-~$ sudo apt install apt-transport-https
-~$ sudo apt update
-~$ sudo apt install dotnet-sdk-3.1
-~$ dotnet --version
-3.1.302
+# Add Microsoft package repository
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+rm packages-microsoft-prod.deb
+
+# Install the SDK
+sudo apt-get update
+sudo apt-get install -y apt-transport-https
+sudo apt-get update
+sudo apt-get install -y dotnet-sdk-6.0
+
+## Check
+dotnet --version
 ```
 
 ##### 2. Build NBXplorer

--- a/docs/Development/Altcoins.md
+++ b/docs/Development/Altcoins.md
@@ -15,7 +15,7 @@ The steps for having a coin added are:
 
 When testing your coin, **DO NOT USE `build.sh`**, since it uses a pre-built docker image.
 
-Instead, install [.NET Core 3.1 SDK](https://www.microsoft.com/net/download/windows) and run:
+Instead, install [.NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) and run:
 
 ```bash
 BTCPAYGEN_CRYPTO1="EXAMPLE-COIN"

--- a/docs/Development/LocalDev.md
+++ b/docs/Development/LocalDev.md
@@ -4,7 +4,7 @@
 
 For the **development environment** you need to install these tools:
 
-* [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download)
+* [.NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
 * Docker: [Windows](https://docs.docker.com/docker-for-windows/install/) | [Mac OS](https://docs.docker.com/docker-for-mac/install/) | [Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
 
 ## Dependencies

